std/exception.d

```diff
@@ -1666,7 +1719,8 @@ class ErrnoException : Exception
         errorHandler.
 +/
 //lazy version
-CommonType!(T1, T2) ifThrown(E : Throwable = Exception, T1, T2)(lazy scope T1 expression, lazy scope T2 errorHandler)
+_std.traits.CommonType!(T1, T2) ifThrown(E : Throwable = Exception, T1, T2)(lazy scope T1 expression,
```

Comment:
I might write this as am explicit eponymous template instead.

Reply:
@dnadlinger wie meinst du das?